setup.py

```diff
@@ -177,6 +177,7 @@ def run(self):
     deps["torchvision"],
     deps["timm"],
     deps["scikit-image"],
+    deps["mediapipe"],
```

Comment:
You can look through the history here and it was requested for mediapipe to not be an explicit dependency.  The detector works if it is installed, and the rest of the library works if it is not.

Reply:
Okay, my bad!